test/TEST-86-MULTI-PROFILE-UKI/Makefile

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+all setup run clean clean-again:
+	true
```

Comment:
This is copied from TEST-85-NETWORK ? It is a kind of fake file to make the test skipped on CentOS CIs. Please create a symlink to TEST-01-BASIC/Makefile . Otherwise, the test will not be run on CentOS CIs.